src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -783,6 +783,15 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 				case IntrinsicId.Type_GetType: {
 						reflectionContext.AnalyzingPattern ();
 
+						var parameters = calledMethod.Parameters;
+						if ((parameters.Count == 3 && (methodParams[2].AsConstInt () == null || methodParams[2].AsConstInt () != 0)) ||
+									(parameters.Count == 5 && (methodParams[4].AsConstInt () == null || methodParams[4].AsConstInt () != 0))) {
+							_context.LogWarning ($"Reflection call '{calledMethod.GetDisplayName ()}' inside " +
```

Comment:
This should use `reflectionContext.RecordUnrecognizedPattern` which does extra things.